CRM/Utils/Time.php

```diff
@@ -123,44 +123,48 @@ public static function getTimeRaw() {
    *   - 'natural' (time moves naturally)
    *   - 'linear:XXX' (time moves in increments of XXX milliseconds - with every lookup)
    *   - 'prng:XXX' (time moves by random increments, between 0 and XXX milliseconds)
+   *
    * @return string
    */
-  public static function setTime($newDateTime, $returnFormat = 'YmdHis') {
+  public static function setTime(int|string $newTime, $returnFormat = 'YmdHis') {
```

Comment:
Oooh. Union types are allowed in PHP _8.0_. 😄  I'm so used to being on `$ancient` PHP, it's hard to remember that sometimes we can use pretty things...